### DISCUSSION TITLE
[@types/firefox-webext-browser] proxy.onRequest.addListener listener should return Promise<ProxyInfo>

### DISCUSSION THREAD

**Discussion by donovanglover** [DefinitelyTyped/DefinitelyTyped#69507] [Issues with a @types package]
> The listener can return any of:
> 
> - a [`proxy.ProxyInfo`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/ProxyInfo) object.
> - an array of [`proxy.ProxyInfo`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/ProxyInfo) objects.
> - a `Promise` that resolves to a `ProxyInfo` object.
> - a `Promise` that resolves to an array of `ProxyInfo` objects.

Docs: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/onRequest#addlistener_syntax

[a comment by an automated account is not shown]

**Comment by jsmnbom** [comment]
Could you post at https://github.com/jsmnbom/definitelytyped-firefox-webext-browser/issues instead pls?
